integration-tests/environments/react-native/ios/Podfile

```diff
@@ -66,8 +66,6 @@ target 'RealmReactNativeTests' do
           # Applying https://github.com/facebook/folly/issues/1470#issuecomment-943123653
           config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] ||= ['$(inherited)', 'FOLLY_HAVE_CLOCK_GETTIME=1']
         end
-        config.build_settings['CC'] = "../../../../../scripts/ccache-clang.sh"
-        config.build_settings['CXX'] = "../../../../../scripts/ccache-clang++.sh"
```

Comment:
Why is this removed? It seems valuable for all other native code being built?

Reply:
It was causing problems, which at the moment I can't remember what exactly.  I can try and get it back in

Reply:
I was able to reactivate this.